### PR TITLE
Confusion_matrix summary.

### LINKED ISSUE #236: Confusion Matrix Summary
**What is your feature request related to?**
* Module request

**Please describe the module/problem**
It would be cool if a confusion matrix summary exist

### PR DESCRIPTION
**Pull request by benHeid**
## Related Issues
closes #236 
## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos)
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Did you update the CHANGELOG

## PR review
Anyone in the community is free to review the PR once the tests have passed.

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones to the PR so it can be classified

### CODE REVIEW

File: pywatts/summaries/confusion_matrix.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from pywatts.core.filemanager import FileManager
+
+import xarray as xr
+from pywatts.core.summary_object import SummaryObjectTable
+from sklearn.metrics import confusion_matrix
+
+
+class ConfusionMatrix(BaseSummary):
+    """
+    Summary to calculate the Confusion Matrix
```
Comment (marianturowski) [inline pywatts/summaries/confusion_matrix.py:13]: confusion

**Review by marianturowski — CHANGES_REQUESTED** [review]
Great! Thank you also for this one.

There is only one comment that you might want to change.
